website/docs/plugin/framework/migrating/attributes-blocks/types.mdx

```diff
@@ -0,0 +1,202 @@
+---
+page_title: 'Attribute Types: Migrating from SDKv2 to the Framework'
+description: >-
+  Migrate attribute type from SDKv2 to the plugin Framework
+---
+
+# Attribute Types
+
+An attribute either contains a primitive type, such as an integer or a string, or contains other attributes. Attributes
+that contain other attributes are referred to as nested attributes, and are implemented by populating the
+`NestedAttributes` field on the `tfsdk.Attribute` struct. Refer to
+[Schemas - Attributes](/plugin/framework/schemas#type) in the Framework documentation for details.
+
+This page explains how to migrate a primitive attribute from SDKv2 to the plugin Framework. For an example of
+migrating a nested block to a nested attribute, refer to [Provider Schema](/plugin/framework/migrating/providers#example-1) in the Framework documentation.
+
+## SDKv2
+
+In SDKv2, attribute types are defined by the `Type` field on the attribute's `schema.Schema` struct.
+
+```go
+func resourceExample() *schema.Resource {
+    return &schema.Resource{
+         /* ... */
+        Schema: map[string]*schema.Schema{
+            "bool_example": {
+                Type:    schema.TypeBool,
+                /* ... */
+            },
+			"float64_example": {
+                Type:    schema.TypeFloat,
+                /* ... */
+            },
+			"int64_example": {
+                Type:    schema.TypeInt,
+                /* ... */
+            },
+			"list_example": {
+                Type:    schema.TypeList,
+				Elem:    &schema.Schema{
+			        Type: schema.TypeBool,
+		        },
+                /* ... */
+            },
+			"map_example": {
+                Type:    schema.TypeMap,
+				Elem:    &schema.Schema{
+			        Type: schema.TypeFloat,
+		        },
+                /* ... */
+            },
+			"set_example": {
+                Type:    schema.TypeSet,
+				Elem:    &schema.Schema{
+			        Type: schema.TypeInt,
```

Comment:
Shouldn't `Elem` be indented to the same level as the `Type` on the line above?

Reply:
Fixed